requirements.txt

```diff
@@ -1,6 +1,9 @@
 importlib_metadata
 jsonschema
-pytest>=5.0.0
+pytest>=5.0.0,<6.0.0
+pytest-forked
+pytest-parallel
+pytest-xdist
```

Comment:
`xdist` is need to give you the basic parallel functionality. The other packages seem to build on top of that. If I omitted it, unit tests were failing miserably.